tests/grid/grid_transform_02.cc

```diff
@@ -83,7 +83,7 @@ int main ()
             }
 
   // then compute new point locations and output the result
-  GridTools::laplace_transform  (new_points,tria);
+  GridTools::laplace_transform  (new_points,tria, static_cast<const Function<dim, double> *>(NULL), true);
```

Comment:
maybe it's related to the fact that  the pointer is to templated class `Function<dim>`.